### DISCUSSION TITLE
How to integrate grapesJs with angular 13

### DISCUSSION THREAD

**Discussion by gaurav-kerkar** [GrapesJS/grapesjs#4217] [Q&A]
I am trying to integrate grapesJs with angular 13, can anyone please help

**Comment by GedMarc** [comment]
i'm busy doing this

**Comment by JGrancha** [comment]
Another one here already had it integrated, but since the latest version where the index.d.ts has been added, the following error appears:
![image](https://user-images.githubusercontent.com/5517910/162942260-3ee748b9-51d3-478f-bd39-189170a69f0f.png)

If I go to the node_modules folder and delete the index.d.ts file everything works as before.
@artf do you know what it can be?

**Reply by GedMarc** [reply]
tsc ?

**Reply by JGrancha** [reply]
I don't understand. I have tried to put it in the tsconfig.json file and it doesn't work either, I have tried to create a typings.d.ts file and add it there as a global variable but it tells me that the grapesjs module is already being used.

**Reply by GedMarc** [reply]
https://www.udemy.com/topic/typescript/

**Reply by JGrancha** [reply]
You look kind of funny copying and pasting a link to a typescript course. I know very well what tsc is, what I don't understand is what you mean by your comment. If you are so kind to explain I will be happy to read you.

**Reply by GedMarc** [reply]
ok - .d.ts - schema file - 
if you don't understand, fortunately,  i won't explain it to you - but you can do a typescript basics course - 

Github issues are for issues found, communities are for questions on how things work and possible i-am-the-problem related things, please start with communities before logging or referencing on issues

**Reply by JGrancha** [reply]
And you really think I haven't tried all that? That's why I come here to try to find a solution together. You're a bit stupid and in your house they don't know it. Anyway, thanks for making me lose a little time and show how much you know hahahaha

**Reply by GedMarc** [reply]
if you had this working already in angular 13 you already had a schema file defining the module - hence duplicate modules.
Your patch schema file must get removed for the official one - 

The current base works just fine, 

Thanks for that
